src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/helper/Await.java

```diff
@@ -21,7 +21,10 @@ public static void envelopeCompleted(String zipFileName) {
         await("File " + zipFileName + " should be completed in processor")
             .atMost(100, SECONDS)
             .pollInterval(1, SECONDS)
-            .until(() -> isEnvelopeCompleted(zipFileName));
+            .until(() -> {
```

Comment:
it is not simplified.

Reply:
`ProcessorEnvelopeStatusChecker` is simplified.
It had 3 methods, now it has 1.

Reply:
because one method is removed to here, we can keep it there.